Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -67,6 +103,12 @@ public class HTTPClient {
         }
     }
 
+    /// Execute POST request using specified URL.
+    ///
+    /// - parameters:
+    ///     - url: Remote URL.
+    ///     - body: Request body.
+    ///     - deadline: Maximum request duration.
```

Comment:
I wouldn't document a dealine as a duration. I'd maybe write the time when the request must have been completed by or so.

Reply:
fixed, thanks!